src/ui/win_types.h

```diff
@@ -146,9 +146,16 @@ typedef enum {
     WIN_VCARD
 } win_type_t;
 
+typedef enum {
+    WIN_SCROLL_FREE,
```

Comment:
This means neither top nor bottom. Maybe an english native can suggest a better name than "free". Something like "inbetween", "medial", "inner" maybe?

Reply:
Done